templates/component-rust/metadata/snippets/component.txt

```diff
@@ -0,0 +1,6 @@
+[component.{{project-name | kebab_case}}]
+source = "{{ output-path }}/target/wasm32-wasip2/release/{{project-name | snake_case}}.wasm"
```

Comment:
We use wasip1 everywhere else - is p2 stable enough that we should be moving to it?